### PR TITLE
if reservation is full not to show event slot on registartion form

### PR DESCRIPTION
**Pull request by sarath-yerra**

### CODE REVIEW

File: src/Modules/Family/EventSlotsModalComponent.js [hunk 1]
```diff
@@ -55,8 +55,8 @@ const EventSlotsModalComponent = (props) => {
         <Modal.Body>
           <div className="container">
             {eventHour.map((item, index) => (
-              <div className="" key={'ul' + index}>
-                {item.event_slots.map((e, i) => (
+              item.event_slots.filter((e, i) => (e.capacity > 0)).map((e, i) => {
```
Comment (maheshkurma) [inline src/Modules/Family/EventSlotsModalComponent.js:58]: need to filter event_slots if open_slots  are available

so, can u replace `e.capacity > 0` with `open_slots > 0`

Comment (sarath-yerra) [reply src/Modules/Family/EventSlotsModalComponent.js]: Oh Yeah done

**Review by maheshkurma — APPROVED** [review]
Now showing event slot times(in modal) only if open slots available.

**Comment by sarath-yerra** [comment]
Screen Shot 2020-10-05 at 3 49 33 PM

**Comment by sarath-yerra** [comment]
Screen Shot 2020-10-05 at 3 48 40 PM

**Pull request merged**